src/components/Devnet/Devnet.tsx

```diff
@@ -1,43 +1,73 @@
 import { useEffect, useState } from 'react';
 import { getInstance } from '../../fhevmjs';
-
 import './Devnet.css';
 import { Eip1193Provider, getAddress } from 'ethers';
 
-const toHexString = (bytes: Uint8Array) => bytes.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
+const toHexString = (bytes: Uint8Array) =>
+  bytes.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
+
+export type DevnetProps = {
+  account: string;
+  provider: Eip1193Provider;
+};
 
-export type DevnetProps = { account: string; provider: Eip1193Provider };
+const CONTRACT_ADDRESS = getAddress(
```

Comment:
No need for getAddress anymore. This is done in fhevmjs directly

Reply:
getAddress was already used, I noticed it improved how it is printed in the frontend when showing the content of the EIP712 fyi.

Reply:
We can add it in fhevmjs if it's not already the case and you think it's useful.

Reply:
I removed the getAddress in my new commmit